src/gc/impl/conservative/gc.d

```diff
@@ -1746,7 +1762,12 @@ struct Gcx
         assert(pool);
 
         debug(PRINTF) printFreeInfo(&pool.base);
+        pool.pagetable[pn] = B_PAGE;
```

Comment:
This looks like a merge conflict (but pool.mark.set where the same applies as in smallAlloc). Updating the pool (pagetable, freepages) already happens in LargeObjectPool.allocPages().